### PR TITLE
[PyTorch] Fix Kaggle Dog

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*

This PR addresses the issues for kaggle-dog section mentioned at #1775.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_computer-vision/kaggle-dog.md [hunk 1]
```diff
@@ -329,7 +329,7 @@ def evaluate_loss(data_iter, net, devices):
         features, labels = features.to(devices[0]), labels.to(devices[0])
         outputs = net(features)
         l = loss(outputs, labels)
-        l_sum = l.sum()
+        l_sum += l.sum()
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-dog.md:332]: ouch

File: chapter_computer-vision/kaggle-dog.md [hunk 2]
```diff
@@ -439,8 +440,8 @@ train(net, train_iter, valid_iter, num_epochs, lr, wd, devices, lr_period,
 
 ```{.python .input}
 #@tab pytorch
-devices, num_epochs, lr, wd = d2l.try_all_gpus(), 5, 0.001, 1e-4
-lr_period, lr_decay, net = 10, 0.1, get_net(devices)
+devices, num_epochs, lr, wd = d2l.try_all_gpus(), 10, 0.0001, 1e-4
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-dog.md:443]: What did you observe when setting lr=0.005 as in mxnet?

File: chapter_computer-vision/kaggle-dog.md (hunk 1 above)
Comment (AnirudhDagar) [reply chapter_computer-vision/kaggle-dog.md:332]: My bad! Sorry. But now it is fixed :)

File: chapter_computer-vision/kaggle-dog.md (hunk 2 above)
Comment (AnirudhDagar) [reply chapter_computer-vision/kaggle-dog.md:443]: With `lr=0.005` in pytorch, the model just didn't converge. Infact the loss kept on increasing for a while then started decreasing. The plot was kind of messy. It was not a strictly decreasing function plot.
